README.md

```diff
@@ -59,6 +59,15 @@ EOF
 
 Few scenarios are available as example in the [scenarios](https://github.com/OpenVoiceOS/ovos-installer/tree/main/scanerios) directory of this repository.
 
+## Uninstall
+
+To uninstall Open Voice OS run the installer with the `--uninstall` option. even if you dont run the installer with the `--uninstall` option, the installer will check if ovos is installed and will ask you if you want to uninstall it.
```

Comment:
When using the `-u`/`--uninstall` option, the installer doesn't ask confirmation, it doesn't go at all in the TUI.